lib/ethon/easy/informations.rb

```diff
@@ -46,6 +46,12 @@ module Informations
         # start until the name resolution was completed.
         :namelookup_time => :double,
 
+        # Return the time, in seconds, it took for all redirection steps
+        # include name lookup, connect, pretransfer and transfer before the
+        # final transaction was started. time_redirect shows the complete
+        # execution time for multiple redirections. (Added in 7.12.3)
```

Comment:
Weird, I took it from my `man curl`:
```
> man curl | grep -A 1 time_redirect
time_redirect  The  time, in seconds, it took for all redirection steps include name lookup, connect, pretransfer and transfer
               before the final transaction was started. time_redirect shows the complete execution time for multiple redirec‐
               tions. (Added in 7.12.3)
```
Maybe the command line formatting option was added after the internal feature.